packages/legend-art/src/components/panel/Panel.tsx

```diff
@@ -56,6 +56,18 @@ export const PanelHeader: React.FC<{
   );
 };
 
+export const Badge: React.FC<{
```

Comment:
It's fine for now, but let's have this in its own separate file and separate the styling when possible! Or else, we call this `PanelHeaderBadge` or something, not sure :D

Reply:
I think it can stand on its own tho it looks kinda weak! Separated file 👍